### ISSUE TITLE
Tidy asset interaction logic

### ISSUE THREAD

**Issue by IanMayo** [serge-web/serge#2058]
I just saw this code that controls whether an asset is interactive or not:
![image](https://user-images.githubusercontent.com/1108513/201468778-35f3a936-f64c-4c35-9c06-e3b25a109299.png)

Is there a reason why we don't do it like this?
```
        {
          assets.map((asset: AssetRow, index: number) => {
            const markerOption = getMarkerOption(asset, index)

           if (interactive) {
            return <Marker {...markerOption} >
              <Tooltip>{asset.name}</Tooltip>
            </Marker>
          }) else {
            return <Marker {...markerOption} pmIgnore interactive={false} >
              <Tooltip>{asset.name}</Tooltip>
            </Marker>
         }
        }
```

**Comment by lebaphi** [comment]
Hi @IanMayo, have you tried with your approach to see if it works?

**Comment by IanMayo** [comment]
No, I was just checking if I was missing something.

**Comment by lebaphi** [comment]
Yep, I've tried but it doesn't seem to work, I found that the Marker only receives the value of the interactive property once when it is initialized, the Marker does not update itself when this value changes. It's like being initialized once then added to the map. So my solution is to initialize 2 Markers with 2 modes: interactive=false and interactive=true as default. So the code doesn't look clean but it works
